pysr/julia_helpers.py

```diff
@@ -64,28 +70,13 @@ def install(julia_project=None, quiet=False):  # pragma: no cover
 
     _version_assertion()
     # Set JULIA_PROJECT so that we install in the pysr environment
-    julia_project, is_shared = _process_julia_project(julia_project)
-    _set_julia_project_env(julia_project, is_shared)
+    processed_julia_project, is_shared = _process_julia_project(julia_project)
+    _set_julia_project_env(processed_julia_project, is_shared)
 
     julia.install(quiet=quiet)
+    Main = init_julia(julia_project, quiet=quiet)
+    io_arg = _get_io_arg(quiet)
 
-    if is_shared:
-        # is_shared is only true if the julia_project arg was None
-        # See _process_julia_project
-        Main = init_julia(None)
-    else:
-        Main = init_julia(julia_project)
-
-    Main.eval("using Pkg")
```

Comment:
`PackageSpec` is not imported because you removed this line.

Reply:
Thanks!